src/node.zig

```diff
@@ -54,14 +54,13 @@ fn init(env: c.napi_env, info: c.napi_callback_info) callconv(.C) c.napi_value {
     }) catch return null;
 
     const cluster = translate.u128_from_object(env, args[0], "cluster_id") catch return null;
-    const concurrency = translate.u32_from_object(env, args[0], "concurrency") catch return null;
     const addresses = translate.slice_from_object(
         env,
         args[0],
         "replica_addresses",
     ) catch return null;
 
-    return create(env, cluster, concurrency, addresses) catch null;
+    return create(env, cluster, addresses) catch null;
 }
 
 fn deinit(env: c.napi_env, info: c.napi_callback_info) callconv(.C) c.napi_value {
```

Comment:
Add a comment on `destroy` explaining that the single-thread runtime does not require synchronization on closing the client.